kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -236,7 +269,24 @@ public void testSignalProcessInstance() throws Exception {
             processClient.abortProcessInstance(CONTAINER_ID, processInstanceId);
             fail(e.getMessage());
         }
+    }
+    
+    @Test
+    public void testSignalProcessInstanceWrongContainer() throws Exception {
+        Long processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_SIGNAL_PROCESS);
+
+        assertNotNull(processInstanceId);
+        assertTrue(processInstanceId.longValue() > 0);
```

Comment:
Unnecessary unboxing
```suggestion
        assertTrue(processInstanceId > 0);
```